server/catgenome/src/main/java/com/epam/catgenome/manager/blast/BlastTaskManager.java

```diff
@@ -93,40 +99,58 @@ private void loadTaskParameters(final BlastTask blastTask) {
     }
 
     @Transactional(propagation = Propagation.REQUIRED)
-    public BlastTask create(final BlastTask blastTask) throws BlastRequestException {
+    public BlastTask create(final TaskVO taskVO) throws BlastRequestException {
+        Assert.assertFalse(CollectionUtils.isEmpty(taskVO.getOrganisms())
```

Comment:
Sorry, my first comment was wrong. We should check `CollectionUtils.isNotEmpty(taskVO.getOrganisms()) && CollectionUtils.isNotEmpty(taskVO. getExcludedOrganisms())`, in this case we'll throw an error when both collections are provided